### PR TITLE
fixed collapsing/expanding bug in displayShow.mako.

### PR DESCRIPTION
**Pull request by p0psicles**
Rewritten the JS so it fits in core.js.

The regex in js is not the prettiest. Could also add like a data-ref-to-button or whatever. But then had to also make changes to templates.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: gui/slick/views/displayShow.mako [hunk 1]
```diff
@@ -414,7 +394,7 @@
             % endif
     </tbody>
         % if sickbeard.DISPLAY_ALL_SEASONS is False:
-        <tbody class="collapse${("", " in")[curSeason == -1]}" id="collapseSeason-${epResult['season']}">
+        <tbody class="toggle collapse${("", " in")[curSeason == -1]}" id="collapseSeason-${epResult['season']}">
```
Comment (p0psicles) [inline gui/slick/views/displayShow.mako:397]: added the toggle as a reference for the jquery selector.

**Comment by fernandog** [comment]
thanks! this fixes the "Uncaught ReferenceError: $ is not defined"

**Comment by miigotu** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/SickRage/SickRage/pull-request/802/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=SickRage/SickRage)

**Pull request merged**